docs/resources/llama-stack-spec.yaml

```diff
@@ -3209,7 +3216,14 @@ paths:
       - description: JSON-encoded provider data which will be made available to the
           adapter servicing the API
         in: header
-        name: X-LlamaStack-ProviderData
+        name: X-LlamaStack-Provider-Data
```

Comment:
Note: update this line: https://github.com/fairinternal/llama-oss-operations/blob/0f95ae40e23d2ab7665bfdac87f2080284f0af8c/scripts/stainless_sync.sh#L69